layouts/partials/head.html

```diff
@@ -49,11 +49,7 @@
 {{ partial "hooks/head-end.html" . }}
 <!--To comply with GDPR, cookie consent scripts places in head-end must execute before Google Analytics is enabled -->
 {{ if hugo.IsProduction }}
-  {{ if hasPrefix .Site.GoogleAnalytics "G-"}}
-    {{ template "_internal/google_analytics.html" . }}
-  {{ else }}
-    {{ template "_internal/google_analytics_async.html" . }}
-  {{ end }}
+  {{ template "_internal/google_analytics.html" . }}
 
   <!-- Google Tag Manager -->
   <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
@cjyabraham - is GTM still being used by this site?

Reply:
yes